app.config.js

```diff
@@ -0,0 +1,12 @@
+import { sentryApi } from './src/config/secrets';
+
+export default {
+  hooks: {
+    postPublish: [
+      sentryApi?.config && {
+        file: 'sentry-expo/upload-sourcemaps',
+        config: sentryApi.config
+      }
+    ]
+  }
+};
```

Comment:
with the new `app.config.js` my Expo Go app lists this project in "recently opened" now as `mobile-app` and not `Smart Village App` anymore???!??!???

Reply:
nice, i wanted to hear this